test/app/app.lua

```diff
@@ -7,4 +7,8 @@ box.cfg{
     wal_max_size        = 2500
 }
 
+local fiber = require('fiber')
+fiber.set_default_deadline(10000)
+fiber.self():set_deadline(10000)
```

Comment:
This looks ugly. `set_default_slice` should affect all fibers for which `set_slice` wasn't called.

Rationale: Suppose the user started a bunch of worker fibers that catch the "slice exceeded" exception and return an error to a client in this case. Suddenly, client start to see errors. The user wants to increase the timeout for all fibers. Currently, it's impossible without iterating over the fibers, which is inconvenient. The user would like to use `set_default_slice` in this case.

Reply:
Added this behavior.